lib/slack.ml

```diff
@@ -45,7 +45,18 @@ let markdown_text_attachment ~footer markdown_body =
     };
   ]
 
-let make_message ?username ?text ?attachments ?blocks ?thread ?handler ?(reply_broadcast = false) ~channel () =
+type msg_followup = {
+  text : string option;
+  attachments : message_attachment list option;
+  blocks : message_block list option;
+  reply_broadcast : bool;
+}
+
+let make_followup ?text ?attachments ?blocks ?(reply_broadcast = false) () =
+  { text; attachments; blocks; reply_broadcast }
```

Comment:
logs should go in snippets, otherwise it will make for giant messages, no? (and in any case we wouldn't want to spam the channel with logs)
```suggestion
let make_followup ?text ?attachments ?blocks () =
  { text; attachments; blocks; reply_broadcast = false }
```

Reply:
I did not know about snippet, I checked them out, I agree they should be used.